dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -2448,11 +2448,11 @@ SimpleColumn* buildSimpleColFromDerivedTable(gp_walk_info& gwi, Item_field* ifp)
                     sc->timeZone(gwi.thd->variables.time_zone->get_name()->ptr());
                     if (!viewName.empty())
                     {
-                            sc->viewName(viewName);
+                            sc->viewName(lower(viewName));
```

Comment:
Do we need the TODO comment here as well, just as you added for the develop PR?

Reply:
Probably not. I sincerely doubt we'll backport MCOL-4144 to 1.4